app/api/webhook/login/route.ts

```diff
@@ -1,15 +1,23 @@
 import axios from "axios";
 import { NextRequest, NextResponse } from "next/server";
 
+function isDev(req: NextRequest) {
+  let dev = false;
+  if (!req.url.includes("sig.mingdao.edu.tw")) {
+    dev = true;
+  }
+
+  return dev;
+}
```

Comment:
To improve security:
- use URL object to parse url.
- use whitelist instead of blacklist.

```ts
function isDev(req: NextRequest) {
  const urlObj = new URL(req.url);

  const devHostnameList = ["sig-beta.zeabur.app", "beta.sig.school"];

  return devHostnameList.includes(urlObj.hostname);
}
```